app/helpers/issues_helper.rb

```diff
@@ -62,7 +62,7 @@ def issue_timestamp(issue)
       ts << capture_haml do
         haml_tag :span do
           haml_concat '&middot;'
-          haml_concat '<i class="fa fa-edit" title="edited"></i> '
+          haml_concat icon('edit', title: "edited")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.